style.css

```diff
@@ -0,0 +1,308 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.container {
+  display: grid;
+  grid-template:
+    ".   left__menu   main__content     right__menu   .  " 1000px /
+    100px 250px 1fr 250px 100px;
+  gap: 5px;
+}
+
+/* LEFT MENU*/
+.left__menu {
+  grid-area: left__menu;
+  display: grid;
+  gap: 20px;
+  background-color: white;
+  grid-template-rows: 350px 40px 60px;
+  position: fixed;
+  left: 150px;
+}
+
+.left__menu .icons {
+  margin-top: 20px;
+}
+
+.left__menu .icons ul li {
+  list-style: none;
+  line-height: 35px;
+}
+
+.left__menu a {
+  text-decoration: none;
+  color: black;
+  font-size: 18px;
+}
+
+.left__menu .icons a.home {
+  color: rgb(94, 147, 221);
+}
+
+.left__menu a:hover {
+  color: rgb(94, 147, 221);
+}
+
+.left__menu svg:hover {
+  color: rgb(94, 147, 221);
+}
+
+.left__menu .tweet__box {
+  background-color: rgb(94, 147, 221);
+  color: white;
+  display: grid;
+  align-items: center;
+  justify-items: center;
+  border-radius: 20px;
+  width: 150px;
+  margin-left: 20px;
+}
+
+@media screen and (min-width: 80rem) {
+  .main__content {
+    margin: 1em 2em;
+  }
+}
+
+/* MAIN CONTAINER */
+.main__content {
+  grid-area: main__content;
+  display: grid;
+  border-right: 2px solid lightgray;
+  border-left: 2px solid lightgray;
+  gap: 20px;
+  grid-template-rows: 150px 1fr;
+  overflow: scroll;
+  scroll-behavior: smooth;
+}
+
+.main__cont .header__content {
+  display: grid;
+  grid-template-rows: 40px 1fr 40px;
+}
+
+.main__content .header__content .main__home {
+  display: grid;
+  position: relative;
+  border-bottom: lightgray 1px solid;
+  align-items: center;
+}
+
+.main__home a {
+  font-size: 17px;
+  padding: 5px;
+  text-decoration: none;
+  color: black;
+  font-weight: bold;
+}
+
+.main__home svg {
+  width: 20px;
+  height: 20px;
+  position: absolute;
+  border-radius: 50%;
+  top: 10px;
+  right: 5px;
+}
+
+.main__content .header__content .happening__message {
+  display: grid;
+  grid-template-columns: 30px 1fr;
+  align-items: center;
+}
+
+.main__content .header__content .happening__message img {
+  border-radius: 50%;
+  width: 40px;
+  height: 30px;
+  padding: 10px;
+}
+
+.main__content .header__content .whats__happening {
+  padding-left: 40px;
+}
+.header__content .icons {
+  display: grid;
+  grid-template-columns: 4fr 1fr;
+  gap: 20px;
+  padding-left: 50px;
+  justify-items: space-between;
+  display: grid;
+}
+
+.header__content .icons .part__one {
+  display: inline;
+  padding: 5px;
+}
+
+.header__content .icons .part__one a {
+  padding: 10px;
+}
+
+.main__content .header__content .icons .part__two {
+  padding: 10px;
+  background-color: rgb(118, 169, 234);
+  justify-content: center;
+  display: grid;
+  border-radius: 20px;
+  color: white;
+  width: 70px;
+  height: 15px;
+  margin-top: 3px;
+  border-bottom: 2px solid lightgray;
+}
+
+.main__content .main__tweet {
+  display: grid;
+  border-top: lightgray solid 1px;
+  grid-template-rows: repeat(auto-fit, 1fr);
```

Comment:
Cool! I'm guessing that's to allow for any number of posts to be added to the feed? Nice feature!